### PR TITLE
Update Version definition.

### PR DESCRIPTION
**Pull request by JessieF**
RDP FileServer PTM SMB

### CODE REVIEW

File: AssemblyInfo/SharedAssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```
Comment (yazeng) [inline AssemblyInfo/SharedAssemblyInfo.cs]: Change that

File: TestSuites/RDP/src/build.cmd [hunk 2]
```diff
@@ -40,9 +40,17 @@ if not defined vspath (
...
+
+
+::Get build version from AssemblyInfo
+set path=%TestSuiteRoot%AssemblyInfo\SharedAssemblyInfo.cs
+set FindExe="%SystemRoot%\system32\findstr.exe"
+set versionStr="[assembly: AssemblyVersion("1.0.0.0")]"
+for /f "delims=" %%i in ('""%FindExe%" "AssemblyVersion" "%path%""') do set versionStr=%%i
+set TESTSUITE_VERSION=%versionStr:~28,-3%
+
+%buildtool% "%TestSuiteRoot%TestSuites\RDP\src\RDP_Client.sln" /t:clean;rebuild
```
Comment (yazeng) [inline TestSuites/RDP/src/build.cmd]: remove this line

File: TestSuites/MS-SMB/src/build.cmd [hunk 3]
```diff
@@ -45,9 +45,16 @@ if not defined vspath (
...
+set TestSuiteRoot=%CurrentPath%..\..\..\
+
+::Get build version from AssemblyInfo
+set path=%TestSuiteRoot%AssemblyInfo\SharedAssemblyInfo.cs
+set FindExe="%SystemRoot%\system32\findstr.exe"
+set versionStr="[assembly: AssemblyVersion("1.0.0.0")]"
+for /f "delims=" %%i in ('""%FindExe%" "AssemblyVersion" "%path%""') do set versionStr=%%i
+set TESTSUITE_VERSION=%versionStr:~28,-3%
+
+%buildtool% "%TestSuiteRoot%TestSuites\MS-SMB\src\MS-SMB_Server.sln" /t:clean
```
Comment (yazeng) [inline TestSuites/MS-SMB/src/build.cmd]: remove this line

**Comment by JessieF** [comment]
RDP FileServer SMB PTM

**Pull request merged**